git-find-lfs-extensions.py

```diff
@@ -111,3 +123,10 @@ def print_line(ext, share_large, count_large, count_all, size_all, min, max):
             int(result[ext]['min']),
             int(result[ext]['max'])
         )
+
+print("\nAdd to .gitattributes:\n")
+for ext in sorted(result, key=lambda x: (result[x]['type'], x)):
+    if len(ext) > 0 and result[ext]['type'] == "binary" and result[ext]['count_large'] > 0:
+        print('*.{} filter=lfs diff=lfs merge=lfs -text'.format(
+            "".join(c if ('0' <= c <= '9') else "[" + c.upper() + c.lower() + "]" for c in ext)
+        ))
```

Comment:
That took a long time to get to work (with google's custom domain email hosting) and in the end it managed to drop the "in-reply-to" header somehow. Google called git-send-email an "insecure app", but that probably would have been a lot easier.

Hopefully it doesn't piss anybody off too much though :)